src/kudu/consensus/consensus.proto

```diff
@@ -399,11 +406,19 @@ message ConsensusRequestPB {
   // UUID of server this request is addressed to.
   optional bytes dest_uuid = 7;
 
+  // UUID of server intended to proxy this request.
```

Comment:
wording is confusing.  "UUID of server intended to send this proxy request to."

Reply:
Hmm, I think the original wording is clearer than the suggestion.